packages/protobuf/src/utils.ts

```diff
@@ -40,22 +40,30 @@ export function parseConfigure(data: protobuf.INamespace) {
 export const createMessageFromName = (messages: protobuf.Root, name: string) => {
     const Message = messages.lookupType(name);
     const MessageType = messages.lookupEnum('MessageType');
-    let messageType = MessageType.values[`MessageType_${name}`];
+    let messageTypeId = MessageType.values[`MessageType_${name}`];
 
-    if (!messageType && Message.options) {
-        messageType = Message.options['(wire_type)'];
```

Comment:
just trying to recall what this was for? 🤔 it looks like it works even without these two lines (at least basic discovery)

Reply:
i guess it's similar case to what im adding, if the message doesnt have defined id in `MessageType_x` it fallsback to message.options (see messages.test.ts)